tpgtools/serialization_helpers.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	cloudresourcemanager "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/cloudresourcemanager"
+	cloudresourcemanagerBeta "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/cloudresourcemanager/beta"
+)
+
+func serializeBetaProjectToHCL(r cloudresourcemanagerBeta.Project) (string, error) {
+	b, err := json.Marshal(r)
+	if err != nil {
+		return "", err
+	}
+	var m map[string]interface{}
+	if err := json.Unmarshal(b, &m); err != nil {
+		return "", err
+	}
+	return serializeProjectToHCL(m)
+}
+
+
+func serializeGAProjectToHCL(r cloudresourcemanager.Project) (string, error) {
+	b, err := json.Marshal(r)
+	if err != nil {
+		return "", err
+	}
+	var m map[string]interface{}
+	if err := json.Unmarshal(b, &m); err != nil {
+		return "", err
+	}
+	return serializeProjectToHCL(m)
+}
+
+func serializeProjectToHCL(m map[string]interface{}) (string, error) {
```

Comment:
I notice this doesn't have folder, is that a git history mishap or is that on purpose?

Reply:
Folder does not need a custom serializer because it has the same schema in DCL and terraform.